JS/companyscript.js

```diff
@@ -0,0 +1,86 @@
+window.onload = () => {
+  const companyImage = document.getElementById("companyImage");
+  const companyName = document.getElementById("companyName");
+  const companyDescription = document.getElementById("companyDescription");
+  const companyLink = document.getElementById("companyLink");
+  const companyPrice = document.getElementById("price");
+  const companyChangesInPrice = document.getElementById("changesInPrice");
+
+  let params = new URL(document.location).searchParams;
+  let symbol = params.get("symbol");
+
+  const urlCompanyProfile = `https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/company/profile/${symbol}`;
```

Comment:
Instead of rewriting the API URL ("https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/) you should use (declare) baseUrl variable. Remember the DRY principle.